addons/mail/static/tests/discuss/core/public_web/sub_channels.test.js

```diff
@@ -192,3 +193,89 @@ test("mention suggestions in thread match channel restrictions", async () => {
     await contains(".o-mail-Composer-suggestion", { text: "Mitchell Admin" });
     await contains(".o-mail-Composer-suggestion", { text: "p1" });
 });
+
+test("sub-thread is muted (opacity-50) when mute is active", async () => {
```

Comment:
```suggestion
test("sub-thread has the right styling when mute is active", async () => {
```

Reply:
Disagree with this change, "has the right styling" is obscure and requires reading the test to understand what this means. `opacity-50` gives a much better grasp of the intent of test with just reading the test name.

Reply:
But the styling could possibly change in the future, and I think the main purpose of the test is checking the styling. By looking at the title, we understand that the if the styling is correct then it is ok. And the styling for now is `opacity-50` but can be other values in the future and we don't need to adjust the title (which states the purpose) of the test.

Reply:
We could word `is visibly muted` rather than `(opacity-50)`, but that's nitpicking and doesn't matter much as both are good enough ;)

Reply:
Checking `.opacity-50` is good enough. Visibly muted for the end-user is reduced opacity so this matches what the user actually sees.

If we change our classnames and style, then this test will likely fail due to removal / change of classname. And if this isn't the case, this is more dead code. I think you are over-thinking in problem of mentioning `.opacity-50` or equivalent there, alternatives make test obscure and this is definitely worst for code maintenance. ;)